sv.c

```diff
@@ -11749,7 +11749,7 @@ C<sv_vcatpvfn_flags> is the most general, and all the other forms are
 implemented by eventually calling it.
 
 It has two sets of argument lists, only one of which is used in any given call.
-The first set, C<args>, is an encapsulated argument list of pointers to C
+The first set, C<args>, is a C<va_list> of pointers to C
 strings.  If it is NULL, the other list, C<svargs>, is used; it is an array
```

Comment:
This should probably say it's a pointer to a `va_list` (which it is).  Testing whether a `va_list` is NULL doesn't make much sense, but does for a pointer.

Reply:
Technically, they all are pointers to `va_list`, but I found two cases besides this one where it made sense to explicitly say so.